mernarezk/assignment1/Q1.java

```diff
@@ -0,0 +1,54 @@
+package Assignment1;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Q1 {
+	/**
+	 * Main Method for testing purposes.
+	 */
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+
+		String s = in.nextLine();
+		String t = in.nextLine();
+		System.out.println(isAnagram(s, t));
+
+	}
+	/**
+	 * Method that checks if two strings are anagrams of the others,
+	 * Cases are ignored ( Cat is anagram of cat ) also any other non english
+	 * letter is ignored.
+	 * @param s first string
+	 * @param t second string
+	 */
+	public static boolean isAnagram(String s, String t) {

```

Comment:
optional: Do you think you could add a boolean parameter to the method which would enable to do a case sensitive check? isAnagram("cat", "Cat", false) would return true, and isAnagram("cat", "Cat", true) would return false.

Reply:
Okay added JUnit test class